doc/publishing-api-syntactic-usage.md

```diff
@@ -3,7 +3,7 @@
 This is the primary interface from publishing apps to the publishing pipeline. Applications PUT items as JSON conforming to a schema specified in govuk-content-schemas.
 Content paths are arbitrated internally by the Publishing API, the content is then forwarded to the live and draft content stores, and placed on the message queue for other apps (eg email-alert-service) to consume.
 
-### Optimistic locking (`previous_version`)
+### [Optimistic locking (`previous_version`)](#optimistic-locking)
```

Comment:
Still not quite sure how GitHub markdown and anchors interact, but I've made some changes in 479e9c7